webserver/grpc/src/main/java/io/helidon/webserver/grpc/ResponseHelper.java

```diff
@@ -27,12 +27,17 @@
 import java.util.function.Supplier;
 import java.util.stream.Stream;
 
+import io.helidon.grpc.core.SafeStreamObserver;
+
 import io.grpc.stub.StreamObserver;
 
 /**
- * A number of helper methods to handle sending responses to a {@link io.grpc.stub.StreamObserver}.
+ * This class now lives here {@link io.helidon.grpc.core.ResponseHelper}.
+ * Will be removed from this location in a future version.
```

Comment:
Please add (to all newly deprecated things) a javadoc tag `@deprecated`, and if marked for removal, add a reference to the new one, such as:
```
@deprecated will be removed in a future version, please use {@link io.helidon.grpc.core.ResponseHelper} instead
```